unit/CMakeLists.txt

```diff
@@ -84,6 +84,7 @@ target_link_libraries(
         goto-checker
         goto-programs
         goto-instrument-lib
+        goto-synthesizer-lib
```

Comment:
This really suggests that you might want to fold the two PRs into the same commit.